sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -38,7 +38,7 @@
 bool is_device_operational = true;
 
 // * PnP Values *
-// The model id is the JSON document (also called the Digital Twins Model Identifier or DTMI) which
+// The model id is the JSON document (also called the Digital s Model Identifier or DTMI) which
 // defines the capability of your device. The functionality of the device should match what is
```

Comment:
`Digital s Model` - not sure what the `s` means here?

Reply:
ahh yea find/replace error. thanks